### DISCUSSION TITLE
7 kyu katas give me two points instead of three?

### DISCUSSION THREAD

**Discussion by sanberkzulfikar** [codewars/codewars.com#2830] [Q&A]
Hey.

When I solve 7 kyu katas, I get 2 points instead of 3. 

Why would it be?

(See: https://docs.codewars.com/gamification/ranks/)

**Comment by EloiseRosen** [comment]
Honor and rank points are different things. See here for honor: https://docs.codewars.com/gamification/honor/#honor-points
